.github/pull_request_template.md

```diff
@@ -1,20 +1,19 @@
-### 💬 Describe the pull request
-A clear and concise description of what the pull request is about.
+### 💬 Describe the pull request/code changes
+_A clear and concise description of what the pull request is about._
 
-### 🗃️ Issues
-This pull request is related to :
-- #1
-- #2
+### 🎟️ Jira
+_This pull request is related to the following Jira issue(s):_
 
 ### 🔢 To Review
-Steps to review the PR:
-1. Install '...'
-2. Go to '...'
-3. Click on '....'
-4. Scroll down to '....'
```

Comment:
Keep that or at least something similar; steps are too often skipped or unexplained.

Reply:
I agree, but I often delete all the steps to rewrite it. It's asked in the comments on the last lines

Reply:
I agree that we could let that, even if we delete it, this example is a better incentive to write something,

Reply:
So @yannlugrin and @Yago can we agree to find maybe a better example or at least, less points ?

Reply:
Make a proposal :)

Reply:
@yannlugrin Check the proposals on the readme of the PR.

Reply:
I think having 2-3 multipoints on the "step to review" is more interesting, with 1 point, people will tend to add only 1 sentence (my point of view)

Reply:
@WengerK Personally, I still have to delete all the lines to rewrite them afterward bc placeholders like those don't fit my needs at all. Maybe the 1. 2. 3. empty is the best solution so in my case I could reuse it.

Reply:
I agree with with them, 2-3 multipoints is a better incentive to write clear review steps. Plus it's really not that tedious to remove if needed.

Reply:
Ok, but can we agree to have something more fluent or appealing? Usually, I have to remove all the text from the steps. 

Have a look in the body of the PR for some ideas.